### ISSUE TITLE
Robot voices for robots eating

### ISSUE THREAD

**Issue by calben** [scarlethammergames/Sound#1]
We need I'd say two or three clips of robots happily eating resources.

Any suggestions on what they might say @gurshane ?

I liked "Omnomnomnom"

**Comment by gurshane** [comment]
Maybe some robotic voice saying omnomnom for the eater and a siren noise when the killer sees the player.

**Comment by Kully** [comment]
Hi guys. I love this comment system. So cool. I can definitely make those.

**Comment by Kully** [comment]
I can put the files in the music folder when finished, although I'm trying to implement sounds into Unity myself too.

**Comment by calben** [comment]
Sounds good!

I can teach you later how this all integrates through Git into the final project to add the sounds in.

We'll also need sounds for particular objects to make the player soundscape.
If we could do something really cool with that, that'd be awesome.

**Comment by calben** [comment]
Indicating someone's current gravitational state through sound could be interesting, for example.

**Comment by Kully** [comment]
Agreed. Don't worry. If there was a sound award for the competition, we would win hands down.
